browser-test/package.json

```diff
@@ -11,7 +11,7 @@
     "@types/node": "^14.14.31",
     "jest": "^26.6.3",
     "jest-playwright-preset": "^1.5.1",
-    "playwright": "^1.9.2",
+    "playwright": "^1.18.0-alpha-dec-23-2021",
```

Comment:
question (blocking): did anything specific prompt the upgrade? why use an alpha release instead of latest stable?

Reply:
This was an accident, I have reverted.